src/Routers/GlobalConfigRouter.js

```diff
@@ -19,8 +19,19 @@ export class GlobalConfigRouter extends PromiseRouter {
 
   updateGlobalConfig(req) {
     return req.config.database.adaptiveCollection('_GlobalConfig')
-      .then(coll => coll.upsertOne({ _id: 1 }, { $set: req.body }))
-      .then(() => ({ response: { result: true } }));
+      .then(coll => coll.find({ '_id': 1 }, { limit: 1 }))
+      .then(results => {
+        const previousConfig = results && results[0] && results[0].params || {};
```

Comment:
@flovilmart: 
The main problem is that I can't merge nested documents with $set afaik.
The whole `params` object get replaced with the new one.
```req.body:
{"params": {"a":"b"}}
In mongodb:
{"params": {"a":"a", "b":"b"}}
Result:
{"params": {"a":"b"}}
Expected:
{"params": {"a":"b", "b": "b"}}

```